modules/monitor.php

```diff
@@ -31,7 +31,16 @@ public function jetpack_modules_loaded() {
 		Jetpack::enable_module_configurable( $this->module );
 	}
 
+	/**
+	 * Send an API request to check if the monitor is active.
+	 *
+	 * @return mixed
+	 *
+	 * @deprecated 8.9.0 The method is not being used since Jetpack 4.2.0, to be removed.
+	 */
 	public function is_active() {
+		_deprecated_function( __METHOD__, 'jetpack-4.2.0' );
```

Comment:
```
	_deprecated_function( __METHOD__, 'jetpack-8.9.0' );
```

Reply:
Damn, I tried to make it as a suggestion you would just accept :/

Reply:
Good catch 👍 
Fixed now.